src/steamship/agents/service/agent_service.py

```diff
@@ -79,3 +83,51 @@ def run_agent(self, agent: Agent, context: AgentContext):
         for func in context.emit_funcs:
             logging.info(f"Emitting via function: {func.__name__}")
             func(action.output, context.metadata)
+
+    @post("prompt")
+    def prompt(self, prompt: str, **kwargs) -> List[Block]:
```

Comment:
I don't think this is an issue. However, this does have one slight real difference with the REPL hosting. It means that _all_ agents will have an (extra) endpoint.

We probably want to disclose this somewhere obvious. Maybe in the docs for `AgentService`.

Reply:
Done!